src/test/java/core/basesyntax/service/impl/TransactionExecutorTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.impl.BalanceHandler;
+import core.basesyntax.strategy.impl.PurchaseHandler;
+import core.basesyntax.strategy.impl.ReturnHandler;
+import core.basesyntax.strategy.impl.SupplyHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class TransactionExecutorTest {
+    private static final String DEFAULT_VALID_FILE = "src/main/resources/example.csv";
+    private FruitTransactionParserImpl fruitTransactionParser = new FruitTransactionParserImpl();
+    private FileReaderCsv fileReaderCsv = new FileReaderCsv();
+
+    @Test
+    void execute_validData_Ok() {
```

Comment:
this is an end-to-end test, which is nice to have, but we also need unit tests to be able to understand which exact part is broken, in case it is. 
So the unit test for `TransactionExecutor` would be to test if it will process all transactions, or maybe just some of them, and test it's behavior with different params like empty or null list